Dockerfile.template

```diff
@@ -1,37 +1,42 @@
-FROM alpine:3.4
+FROM gcr.io/google_containers/ubuntu-slim:0.4
 MAINTAINER Yusuke KUOKA <ykuoka@gmail.com>
 
-ENV S6_OVERLAY_VERSION=v1.17.2.0
+# Ensure there are enough file descriptors for running Fluentd.
+RUN ulimit -n 65536
```

Comment:
@cw-tomita Thanks for the info!  I will delete this line because k8s controls the limit.